python/cac_tripplanner/templates/home.html

```diff
@@ -62,8 +62,35 @@
     <div class="places">
         <div class="places-content">
             <header class="places-header">
-                <h1>Places we love</h1>
-                <a href="#" class="map-view-btn">Map View</a>
+                <div class="places-header-content">
+                    <h1>Places we love</h1>
+                    <a href="#" class="map-view-btn">Map View</a>
+                    <div class="filter-picker">
+                        <div class="filter-toggle">
+                            <div class="all filter-option on" title="All" data-filter="All">
+                                <span class="filter-label">All</span>
+                            </div>
+                            <div class="events filter-option" title ="Events" data-filter="Events">
+                                <span class="filter-label">Events</span>
+                            </div>
+                            <div class="nature filter-option" title="Nature" data-filter="Nature">
+                                <span class="filter-label">Nature</span>
+                            </div>
+                            <div class="exercise filter-option" title="Exercise"
+                                data-filter="Exercise">
+                                <span class="filter-label">Exercise</span>
+                            </div>
+                            <div class="relax filter-option" title="Relax" data-filter="Relax">
+                                <span class="filter-label">Relax</span>
+                            </div>
+                            <div class="educational filter-option" title ="Educational"
+                                data-filter="Educational">
+                                <span class="filter-label">Educational</span>
+                            </div>
```

Comment:
I'd really like to not have the header section templated. There was a reason for that before, but maybe it can be ripped out.

There's going to be styling dependent on the category list, which is supposed to be preset for that reason. It might just be icons for each though, but that would be information the front-end should have and not just the python template.